recipes/e3sm-unified/meta.yaml

```
@@ -44,18 +44,18 @@ requirements:
     - ipython
     - jupyter
     - livvkit 3.1.0
-    - mache 1.28.0
+    - mache 1.31.0
     - moab 5.5.1 {{ mpi_prefix }}_tempest_*  # [mpi != 'hpc']
-    - mosaic 1.1.0  # [py>=310]
-    - mpas-analysis 1.13.0
-    - mpas_tools 0.36.0
-    - nco 5.3.2  # [mpi != 'hpc']
-    - pcmdi_metrics 3.8.2
+    - mosaic 1.1.0
+    - mpas-analysis 1.14.0rc01
```

```suggestion
    - mpas-analysis 1.14.0rc2
```
https://github.com/conda-forge/mpas-analysis-feedstock/pull/75